### PR TITLE
RHIROS-193 - edit messages for empty states

### PR DESCRIPTION
**Pull request by kgaikwad**
1. Modify empty state message "**No recommendations**
No known recommendations affect this system. "  to "**No recommendations**
This system isn’t affected by any known recommendations."

Screenshot from 2021-07-09 19-03-18

2. Modify message "**Resource optimization is not yet configured**
Activate application in the Service Enablement Dashboard" to "
**Resource optimization isn’t configured yet**
To start configuration, activate this application in the Service Enablement Dashboard.
[Start configuration]"
Screenshot from 2021-07-09 18-01-57

+adding @terezanovotna into loop for UX review.

### CODE REVIEW

**Review by apuntamb — APPROVED** [review]
LGTM!
@terezanovotna any suggestions you'd like to add?

**Comment by terezanovotna** [comment]
Looks great!

File: src/Components/ServiceNotConfigured/ServiceNotConfigured.js [hunk 1]
```diff
@@ -8,8 +8,12 @@ export const ServiceNotConfigured = () => (
         <EmptyState className='empty-state-not-configured'>
             <EmptyStateIcon icon={WrenchIcon} />
             <Title headingLevel="h5" size="lg">
-                Resource optimization is not yet configured
+                Resource optimization isn&apos;t configured yet
             </Title>
+            <EmptyStateBody>
+                To start configuration, activate this application in the Service Enabled Dashboard.
+                <Text>[Start configuration]</Text>
```
Comment (kgaikwad) [inline src/Components/ServiceNotConfigured/ServiceNotConfigured.js:15]: Hey @apuntamb, do not merge this pull-request immediately as I would like to know - whether `[start configuration]` is just a placeholder or any action expected here?
I will confirm this behaviour with rest of the team.

Comment (kgaikwad) [reply src/Components/ServiceNotConfigured/ServiceNotConfigured.js:15]: As per offline discussion, initially ROS will provide PDF link for end-user. 
While GA, needs to link official doc link at [Start configuration].
For now, both are not available so keeping this PR on hold.

**Comment by radekvokal** [comment]
@kgaikwad I will provide you link to documentation when it's available.

**Comment by kgaikwad** [comment]
Screenshot from 2021-08-09 15-26-33

**Comment by terezanovotna** [comment]
Looks great! The grey text is hard to use, can we use darker grey, so it is more easier to read?
Thanks so much!

**Comment by kgaikwad** [comment]
> The grey text is hard to use, can we use darker grey, so it is more easier to read?

@terezanovotna, default css applied is `color: var(--pf-c-empty-state__body--Color) i.e --pf-global--Color--200`

1. with `color: var(--pf-global--Color--300);`
Screenshot from 2021-08-09 20-48-27


2. with `color: var(--pf-global--Color--100);`
Screenshot from 2021-08-09 21-11-31

Please let me know which one looks better than previous.

File: src/Components/ServiceNotConfigured/ServiceNotConfigured.js [hunk 2]
```diff
@@ -1,15 +1,51 @@
...
+                        <StackItem>
+                            1. Prerequisites: insights-client, ansible (Read FAQ if you
+                don&apos;t know how to install it).
+                        </StackItem>
+                        <StackItem>
+                            2. Download and run playbook.
+                        </StackItem>
+                        <StackItem>
+                            <ClipboardCopy>
+                                wget https://raw.githubusercontent.com/RedHatInsights/ros-backend/main/ansible-playbooks/ros_install_and_set_up.yml
```
Comment (pgarciaq) [inline src/Components/ServiceNotConfigured/ServiceNotConfigured.js:30]: "wget" is not installed by default, we should use "curl -O" 

```suggestion
                                curl -O https://raw.githubusercontent.com/RedHatInsights/ros-backend/main/ansible-playbooks/ros_install_and_set_up.yml
```

Comment (pgarciaq) [inline src/Components/ServiceNotConfigured/ServiceNotConfigured.js:30]:
```suggestion
                                curl -O https://raw.githubusercontent.com/RedHatInsights/ros-backend/main/ansible-playbooks/ros_install_and_set_up.yml
```

**Comment by kgaikwad** [comment]
@terezanovotna,  @pgarciaq - Thank you for review.
I have updated the changes as per suggestions. Here is the updated screenshot - 
Screenshot from 2021-08-10 19-57-14

Please request to review updated changes.

**Review by pgarciaq — APPROVED** [review]
LGTM!

**Review by apuntamb — APPROVED** [review]
Looks good!

**Comment by apuntamb** [comment]
Merged! Thanks @kgaikwad @radekvokal @terezanovotna @pgarciaq for the effort! :+1: :ok_hand:

**Pull request merged**